node/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-pipelines-task-lib",
-  "version": "3.0.2-preview",
+  "version": "3.0.3-preview",
```

Comment:
Do we need preview description?

Reply:
Oh I meant the other way :) we have 3.0.3-preview, is there any reason we don't have 3.0.3?

Reply:
I guess since we are doing new major it's probably fine.